docs/03-tutorials/00-application-connectivity/ra-02-maintain-secure-connection-with-compass.md

```diff
@@ -42,7 +42,9 @@ After you have established a secure connection with Compass, you can fetch the c
     openssl genrsa -out compass-app.key $KEY_LENGTH
     openssl req -new -sha256 -out compass-app.csr -key compass-app.key -subj "{SUBJECT}"
     ```
-   > **NOTE:** The key length is configurable, however, 4096 is the recommended value.
+
+   > [!NOTE]
+   > The key length is configurable, however, 4096 is the recommended value.
```

Comment:
The note displays badly on GitHub 
![Uploading Screenshot 2024-01-29 at 10.50.54.png…]()